scriptharness/__init__.py

```diff
@@ -136,7 +144,7 @@ def get_config(name="root"):
         of name `name`.
 
     Returns:
-      config (dict): By default scriptharness.structures.LoggingDict
+      config (Dict[str, str]): By default scriptharness.structures.LoggingDict
```

Comment:
config doesn't have to have str values; they can be dicts, lists, strs, ints... maybe 'object' or 'various' or something?